### PR TITLE
Adds support for running Xenon on a computer, and using Plethora's chest.drop instead

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main.lua [hunk 1]
```diff
@@ -48,6 +44,11 @@ local function xenon()
   --#include "src/sections/updates.lua"
 
   --#include "src/sections/renderer.lua"
+  
+  
+  if not (turtle or layoutMode or config.out_chest) then
```
Comment (incinirate) [inline src/main.lua]: Config convention is camelCase not snake_case

File: src/sections/inventory.lua [hunk 2]
```diff
@@ -128,10 +128,17 @@ local function dispense(mcname, count)
     end
   end
 
-  for i = 1, 16 do
-    if turtle.getItemCount(i) > 0 then
-      turtle.select(i)
-      turtle.drop()
+  if config.out_chest then
+    local oc = peripheral.wrap(config.out_chest)
```
Comment (incinirate) [inline src/sections/inventory.lua]: This should be done in `sections/peripherals.lua`

Comment (incinirate) [reply src/sections/inventory.lua]: Also should be `local outChest`, two letter names are not clear.

**Review by incinirate — CHANGES_REQUESTED** [review]
Have you tested this under the following conditions?

* outChest is the only chest (meaning `config.chest=outChest` and `config.outChest=outChest`)
* outChest pulls from a network

**Comment by incinirate** [comment]
Also the example config needs to be updated

File: examples/config.lua [hunk 3]
```diff
@@ -1,82 +1,86 @@
...
-
-  layout = "layout.html", -- The file from which xenon should load the layout
-  styles = "styles.css",  -- The file from which xenon should load the stylesheet
-
-  host  = "k123456789", -- * The Krist Address to listen to (*Required)
-  name  = "name",     -- * The Krist Name to use for purchases (*Required)
-  pkey  = "",         -- * The private key to use for refunds (*Required)
-  pkeyFormat = "raw", -- Either 'raw' or 'kwallet', defaults to 'raw'
-  -- NOTE: It is not recommended to use kwallet, the best practice is to convert your pkey (using
+	title = "My Shop", -- Name of the shop, for logging purposes only
```
Comment (incinirate) [inline examples/config.lua]: Oh god the tabs have invaded

File: src/sections/inventory.lua [hunk 4]
```diff
@@ -1,149 +1,162 @@
...
-  for k, item in pairs(config.items) do
-    if item.addy == name then
-      return item, util.toListName(item.modid, item.damage or 0, item.predicateID or 0)
-    end
-  end
+	for k, item in pairs(config.items) do
+		if item.addy == name then
+			return item, util.toListName(item.modid, item.damage or 0, item.predicateID or 0)
+		end
+	end
```
Comment (incinirate) [inline src/sections/inventory.lua]: taaaaaaaabs

**Review by incinirate — CHANGES_REQUESTED** [review]
You managed to clobber my indentation

File: examples/config.lua (hunk 3 above)
Comment (ghost) [reply examples/config.lua]: VS Code autoformated. How do I fix it?

File: src/sections/inventory.lua (hunk 4 above)
Comment (ghost) [reply src/sections/inventory.lua]: VS Code autoformated. How do I fix it?

**Comment by ghost** [comment]
Closing to fix indents

**Pull request closed without merging**